src/Hangfire.Core/Dashboard/Pager.cs

```diff
@@ -26,42 +26,57 @@ public class Pager
 
         private int _startPageIndex = 1;
         private int _endPageIndex = 1;
-
-        public Pager(int from, int perPage, long total)
+                
+        public Pager(int from, int perPage, long total, string filterString = null, string startDate = null, string endDate = null)
         {
+            JobsFilterText = string.IsNullOrEmpty(filterString) ? null : filterString;
```

Comment:
I'm not sure if that's a good way of doing it since the setters in the Pager class are declared private. It would break the existing code style.